maxim.timofeev/task-2-1/cmd/service/main.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+)
+
+const (
+	defaultMinTemp = 15
+	defaultMaxTemp = 30
+)
+
+var errInvalidDirection = errors.New("invalid direction")
+
+type conditioner struct {
+	minTemp int
+	maxTemp int
+}
+
+func NewConditioner(minTemp int, maxTemp int) *conditioner {
```

Comment:
чет все в перемешку конструктор гломальный, методы, локальные, давай уж что-нибудь одно... 

```suggestion
func newConditioner(minTemp int, maxTemp int) *conditioner {
```

Reply:
исправил